2017-05-08 17:55:57,154 [82] ERROR LeaRun.DataAccess.DbHelper - 超时时间已到。超时时间已到，但是尚未从池中获取连接。出现这种情况可能是因为所有池连接均在使用，并且达到了最大池大小。
2017-05-08 17:55:57,154 [598] ERROR LeaRun.DataAccess.DbHelper - 超时时间已到。超时时间已到，但是尚未从池中获取连接。出现这种情况可能是因为所有池连接均在使用，并且达到了最大池大小。
2017-05-08 17:56:01,192 [593] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,191 [567] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,191 [587] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,192 [582] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,192 [595] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,191 [563] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,191 [562] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,192 [588] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,192 [581] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,191 [95] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,252 [584] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,267 [586] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,270 [95] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,273 [595] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,273 [562] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,273 [593] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,282 [587] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,318 [644] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,318 [773] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,320 [561] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,322 [578] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
2017-05-08 17:56:01,322 [598] ERROR LeaRun.DataAccess.DbHelper - 已成功与服务器建立连接，但是在登录过程中发生错误。 (provider: TCP Provider, error: 0 - 远程主机强迫关闭了一个现有的连接。)
